scripts/ci/docker/build_docker_visit_ci.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+# Project developers.  See the top-level LICENSE file for dates and other
+# details.  No copyright assignment is required to contribute to VisIt.
+
+#
+# Helper script that drives our docker build and tag for ci
+#
+
+import os
+import sys
+import subprocess
+import shutil
+import datetime
+
+from os.path import join as pjoin
+
+def remove_if_exists(path):
+    """
+    Removes a file system path if it exists.
+    """
+    if os.path.isfile(path):
+        os.remove(path) 
+    if os.path.isdir(path):
+        shutil.rmtree(path)
+
+def timestamp(t=None,sep="-"):
+    """ Creates a timestamp that can easily be included in a filename. """
+    if t is None:
+        t = datetime.datetime.now()
+    sargs = (t.year,t.month,t.day)
+    sbase = "".join(["%04d",sep,"%02d",sep,"%02d"])
+    return  sbase % sargs
+
+def sexe(cmd,ret_output=False,echo = True):
```

Comment:
Is there a tad more descriptive function name?

Reply:
system execute :-). Note that this function exists in 10s to 100s of python scripts I have sprinkled across the universe.
It is already in use in VisIt in a few places as well.  It's unique and easy to type, and it has a doc string :-).